src/stores.ts

```diff
@@ -11,7 +11,8 @@ import {
 import { type RouteInfo } from "./worker";
 
 // A global singleton, containing a loaded map
-// TODO | null. When we enable strictNullChecks, this'll become a problem
+// @ts-ignore TODO By construction, no components using the store should be
```

Comment:
We could make all components using $map handle the possibility of not being loaded. https://github.com/dimfeld/svelte-maplibre/ does this, but it's complex. For now, I think carrying on with the assumption of it being loaded is OK